ts/src/pro/binance.ts

```diff
@@ -1688,8 +1688,11 @@ export default class binance extends binanceRest {
                     parsed['fees'] = fees;
                 }
                 parsed['trades'] = this.safeValue (order, 'trades');
-                parsed['timestamp'] = this.safeInteger (order, 'timestamp');
-                parsed['datetime'] = this.safeString (order, 'datetime');
+                const timestamp = this.safeInteger (parsed, 'timestamp');
```

Comment:
@pcriadoperez sorry I don't fully understand this yet, how can we get an ws update with an outdated timestamp?